### ISSUE TITLE
Chaos / Diablo issues (blizz sorc)

### ISSUE THREAD

**Issue by jik0n** [hectorgimenez/koolo#30]
Doesn't seem to be activating all things to summon Diablo. I managed to capture a video of it happening

https://youtu.be/Q_1iUXbtT6Y?si=tsOAU9qEJOwBDhZb

I start the bot then immediately move my mouse off the VM screen

**Comment by hectorgimenez** [comment]
Yeah, seal bosses detection needs to be improved. I've uploaded few changes that should improve it, not 100% fixes althought, please try.

**Comment by jik0n** [comment]
few test runs I've done have been successful, I'll leave it up to you if you want the issue to remain open until 100% fully resolved. I can monitor the bot more tomorrow for longer

**Comment by hectorgimenez** [comment]
Yeah... that seal has been bugged for 20 years and probably will take other 20 years to fix it. I can try to implement some kind of retry and/or timeout.

**Comment by jik0n** [comment]
Noticing some mobs will be behind a wall and the sorc will sit there attempting to kill them until game timeout or I intervene. Is it possible to implement a time spent on trying to kill mobs? Isn't even a boss or required for the seals just a random misc mob it can't hit

**Comment by hectorgimenez** [comment]
Yeah that's a good point, that area is super buggy. Maybe a timeout and trying to skip non-reachable monsters, like monsters outside of walkable areas. Worth to try
